xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/WCAGUtils.java

```diff
@@ -102,29 +105,40 @@ public void writeWCAGResults() throws IOException
             getWCAGContext().getTestClassName(), totalTime);
 
         File wcagDir = new File(getWCAGReportPathOnHost());
+        Files.createDirectory(wcagDir.toPath());
+        String outputName = "wcagOverview.txt";
```

Comment:
I think this `outputName` would be clearer if replaced with a constant for each file (wcagOverview, wcagIncomplete...).
That way you can just use the right const at the right place instead of overriding the value of a generic variable at difference places.

Reply:
Otherwise, lgtm

Reply:
:+1: Addressed in https://github.com/xwiki/xwiki-platform/pull/2299/commits/e2260656c492e6f74acea7900761ebc76885e923